### PR TITLE
config: add codecov for ti-community-bot

### PR DESCRIPTION
**Pull request by hi-rustin**

### CODE REVIEW

File: prow/jobs/ti-community-infra/ti-community-bot/ti-community-bot-postsubmits.yaml [hunk 1]
```diff
@@ -12,7 +12,16 @@ postsubmits:
               - /bin/bash
               - -c
               - >
+              - >
```
Comment (Mini256) [inline prow/jobs/ti-community-infra/ti-community-bot/ti-community-bot-postsubmits.yaml:15]: There seems to be an extra line here.

Comment (hi-rustin) [reply prow/jobs/ti-community-infra/ti-community-bot/ti-community-bot-postsubmits.yaml]: fixed.

**Pull request merged**